modules/installation-osp-default-kuryr-deployment.adoc

```diff
@@ -26,7 +26,7 @@ Use the following quota to satisfy a default cluster's minimum requirements:
 |vCPUs                   | 28
 |Volume storage          | 275 GB
 |Instances               | 7
-|Security groups         | 250 - 1 needed per Service and per NetworkPolicy
+|Security groups         | 250 - 1 needed per Service (at {rh-openstack} version 16 with amphora driver instead of ovn octavia driver, this belongs to the service account instead of the user project) and per NetworkPolicy
```

Comment:
I'll move this into a note below the table so that it fits.

Reply:
sounds good